src/coreclr/src/jit/codegenlinear.cpp

```diff
@@ -938,6 +938,84 @@ GenTree* sameRegAsDst(GenTree* tree, GenTree*& other /*out*/)
     }
 }
 
+//------------------------------------------------------------------------
+// genUnspillLocal: Reload a register candidate local into a register, if needed.
+//
+// Arguments:
+//     varNum    - The variable number of the local to be spilled.
+//                 It may be a local field.
+//     type      - The type of the local.
+//     lclNode   - The node being spilled. Note that for a multi-reg local,
+//                 the gtLclNum will be that of the parent struct.
+//     regNum    - The register that 'varNum' should be loaded to.
+//     reSpill   - True if it will be immediately spilled after use.
+//     isLastUse - True if this is a last use of 'varNum'.
+//
+void CodeGen::genUnspillLocal(
+    unsigned varNum, var_types type, GenTreeLclVar* lclNode, regNumber regNum, bool reSpill, bool isLastUse)
+{
+    LclVarDsc* varDsc = &compiler->lvaTable[varNum];
+    inst_set_SV_var(lclNode);
+    GetEmitter()->emitIns_R_S(ins_Load(type, compiler->isSIMDTypeLocalAligned(varNum)), emitTypeSize(type), regNum,
```

Comment:
Nit, to avoid a strange new line with `	                              varNum, 0);`
```
    instruction ins = ins_Load(type, compiler->isSIMDTypeLocalAligned(varNum));
    GetEmitter()->emitIns_R_S(ins , emitTypeSize(type), regNum,  varNum, 0);
```